src/clj/ataru/applications/excel_export.clj

```diff
@@ -366,8 +366,8 @@
                     (not (util/answerable? %))
                     (and (not (included-ids (:id %)))
                          (or (and skip-answers?
```

Comment:
Tässä tarpeeton `or` ja sisäkkäiset `and`

Reply:
OK